environment/k8s_environment.go

```diff
@@ -123,26 +126,39 @@ func NewK8sEnvironment(
 	}
 	env.chaos = cc
 
+	return env, nil
+}
+
+// DeploySpecs deploys a all the specs in the array
```

Comment:
_deploys all the specs in the array_

`the array` is not exactly clear as to what we're deploying from. Maybe a better version would be something like 

> deploys all specs in the provided environment init function

>